docs/pages/api.rst

```diff
@@ -0,0 +1,121 @@
+.. _nuts-discovery-api:
+
+***
+Api
+***
+
+The *Nuts Discovery Service* consists of 2 api's: the network map and certificare api's. The certificate api is in the initial setup phase of the Corda node. To publish the node details with an electronic signature and retrieve the signed certificate from the *Nuts Discovery Service*. The network map api is used to retrieve details about all the nodes that are connected to the network.
+
+Certificate API
+===============
+
+The *Certificate API* is part of the Corda node but it's seems to be lacking documentation. After reverse engineering, 2 requests could be identified. The initation process for a Corda node can be started with:
+::
+
+    java -jar corda.jar initial-registration -p changeit
+
+where the given password is the password of the root truststore. More info on how to start and configure a node can be found in :ref:`node setup <nuts-discovery-setup>`.
```

Comment:
The root truststore is a Corda thing right? If so, prefixing it with Corda like `the Corda root truststore` can clarify things.